docs/docs-graphql/schema.graphqls

```diff
@@ -2227,7 +2227,7 @@ type Query {
   ): Query!
 
   """Reference a secret by name."""
-  secret(name: String!): Secret!
+  secret(accessor: String, name: String!, plaintextDigest: String): Secret!
```

Comment:
I genuinely can't see a legitimate reason for users calling this function manually - it exists purely as an implementation detail IMO.

Is there a way that we can hide this from the generate graphql docs + all the SDKs? Seems related to https://github.com/dagger/dagger/pull/6714#discussion_r1500466800

Reply:
This has actually come up before in the wild, not just an internal thing - it was added for the internal call, yes, but in my mind it was also adding an API feature that has been requested for a while:

* https://discord.com/channels/707636530424053791/1200099731045613598/1200099731045613598
* https://discord.com/channels/707636530424053791/1153455796164771920/1153455796164771920
* (I think more but the point has been made)

Does it still work without these arguments?

Reply:
Ah I think we can keep the function itself, but I think it shouldn't be possible to call with accessor/etc manually. I'd be happy with just making them hidden, but...

If we could just make it impossible to do so, then I don't think we'd even need the plaintext digest thing I introduced.